plugins/Monolog/config/config.php

```diff
@@ -97,6 +99,14 @@
     'Piwik\Plugins\Monolog\Handler\FileHandler' => DI\create()
         ->constructor(DI\get('log.file.filename'), DI\get('log.level.file'))
         ->method('setFormatter', DI\get('log.lineMessageFormatter.file')),
+    
+    '\Monolog\Handler\ErrorLogHandler' => DI\autowire()
+        ->constructorParameter('level', DI\get('log.level.errorlog'))
+        ->method('setFormatter', DI\get('log.lineMessageFormatter.file')),
+
+    '\Monolog\Handler\SyslogHandler' => DI\create()
+        ->constructor(DI\get('log.syslog.ident'), 'syslog', DI\get('log.level.syslog'))
```

Comment:
Is this logging to the "syslog" facility? I would guess that "user" is most appropriate, but "syslog" is definitely not correct =)

(Discovered while trying to match on the facility and program name using syslog-ng.)

Lazy reference: https://en.wikipedia.org/wiki/Syslog#Facility

Reply:
You are correct this should be 'user'.